docs/user/masternodes/understanding.rst

```diff
@@ -81,6 +81,25 @@ real-time payment rates, and `this site
 <http://178.254.23.111/~pub/Dash/Dash_Info.html>`_ for various real-time
 statistics on the masternode network.
 
+High-Performance Masternodes (HPMNs)
+------------------------------------
+
+High-Performance Masternodes (HPMNs) are a subset of Masternodes that have 
+been created to host the Dash Platform. An HPMN is a lot like Masternodes 
+with three main differences:
+
+- A dedicated IP address: These usually come with the VPS/server.
+- The collateral to own an HPMN is 4000 Dash as opposed to 1000 Dash 
+for regular masternodes.
+- The recommended specs for HPMNs are higher than those for 
+regular masternodes.
+- HPMNs serve Platform along with Core, while regular masternodes 
+only serve Core.
```

Comment:
This still doesn't render properly as a bulleted list. Also, please see the original comment about changing this to a table instead of a bulleted list: https://github.com/dashpay/docs/pull/197#discussion_r1101740578